packages/portfolio-contract/test/mocks.ts

```diff
@@ -174,36 +174,72 @@ export const makeUSDNIBCTraffic = (
   },
 });
 
-export const axelarChainsMapMock: AxelarChainsMap = {
+export const contractAddressesMock: EVMContractAddressesMap = {
   Ethereum: {
-    caip: 'eip155:1',
-    axelarId: 'Ethereum',
-    contractAddresses: {
-      aavePool: '0x1111111111111111111111111111111111111111',
-      compound: '0xc3d688B66703497DAA19211EEdff47f25384cdc3',
-      factory: '0xef8651dD30cF990A1e831224f2E0996023163A81',
-      usdc: '0xCaC7Ffa82c0f43EBB0FC11FCd32123EcA46626cf',
-    },
+    aavePool: '0x1111111111111111111111111111111111111111',
+    compound: '0x2222222222222222222222222222222222222222',
+    factory: '0x3333333333333333333333333333333333333333',
```

Comment:
to make snapshots easier to read, consider:

```suggestion
    aavePool: '0xAAAA111111111111111111111111111111111111',
    compound: '0xCCCC222222222222222222222222222222222222',
    factory: '0xFFFF333333333333333333333333333333333333',
```

Reply:
Using these suggested addresses result in:
```
Error#1: sendGmp failed: InvalidAddressError: Address "0xCCCC222222222222222222222222222222222222" is invalid.

- Address must be a hex value of 20 bytes (40 hex characters).
- Address must match its checksum counterpart.

Version: viem@2.31.0

- Address must be a hex value of 20 bytes (40 hex characters).
- Address must match its checksum counterpart.

Version: viem@2.31.0
    at sendGmp (file:///Users/rabisiddique/Desktop/Work/agoric-sdk/packages/portfolio-contract/src/pos-gmp.flows.ts:238:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async supplyToCompound (file:///Users/rabisiddique/Desktop/Work/agoric-sdk/packages/portfolio-contract/src/pos-gmp.flows.ts:345:3)
    at async changeGMPPosition (file:///Users/rabisiddique/Desktop/Work/agoric-sdk/packages/portfolio-contract/src/pos-gmp.flows.ts:469:7)
    at async rebalance (file:///Users/rabisiddique/Desktop/Work/agoric-sdk/packages/portfolio-contract/src/portfolio.flows.ts:201:5)
    at async openPortfolio (file:///Users/rabisiddique/Desktop/Work/agoric-sdk/packages/portfolio-contract/src/portfolio.flows.ts:294:9)
    at async Promise.all (index 0)
    at async file:///Users/rabisiddique/Desktop/Work/agoric-sdk/packages/portfolio-contract/test/portfolio.flows.test.ts:495:20

```

Reply:
boring. oh well.